docs/about/contributing/CONTRIBUTING.md

```diff
@@ -148,6 +148,32 @@ channel.
 
 - [Frontend styling guide](./frontend.md#styling)
 
+## Pull Requests
```

Comment:
This section feels like it could absorb https://coder.com/docs/about/contributing/CONTRIBUTING#reviews, and make it more concise.